diff --git a/node_modules/@keystonehq/bc-ur-registry/dist/CryptoAccount.d.ts b/node_modules/@keystonehq/bc-ur-registry/dist/CryptoAccount.d.ts
index 07f92a7..a33a097 100644
--- a/node_modules/@keystonehq/bc-ur-registry/dist/CryptoAccount.d.ts
+++ b/node_modules/@keystonehq/bc-ur-registry/dist/CryptoAccount.d.ts
@@ -1,5 +1,5 @@
 /// <reference types="node" />
-import { CryptoOutput } from '.';
+import { CryptoOutput } from './CryptoOutput';
 import { DataItem } from './lib';
 import { RegistryItem } from './RegistryItem';
 export declare class CryptoAccount extends RegistryItem {
diff --git a/node_modules/@keystonehq/bc-ur-registry/dist/CryptoAccount.js b/node_modules/@keystonehq/bc-ur-registry/dist/CryptoAccount.js
index 81e28db..88b8585 100644
--- a/node_modules/@keystonehq/bc-ur-registry/dist/CryptoAccount.js
+++ b/node_modules/@keystonehq/bc-ur-registry/dist/CryptoAccount.js
@@ -1,7 +1,7 @@
 "use strict";
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.CryptoAccount = void 0;
-const _1 = require(".");
+const _1 = require("./CryptoOutput");
 const lib_1 = require("./lib");
 const RegistryItem_1 = require("./RegistryItem");
 const RegistryType_1 = require("./RegistryType");
